leptonai/api/types.py

```diff
@@ -393,7 +396,8 @@ class LeptonJobSpec(BaseModel):
     parallelism: int = 1
     envs: List[EnvVar] = []
     mounts: List[Mount] = []
-    intra_job_communication: bool = False
+    ttl_seconds_after_finished: Optional[int] = None
+    intra_job_communication: Optional[bool] = None
```

Comment:
Do we want to always set this across the board to true? Note - if worker is defined, this will automatically be set to true.

Reply:
IMO, it's okay for each worker within a job to be able to communicate with each other.

Reply:
It's not like that - I'd like to keep a minimum viable principle and not blindly set it to on. Otherwise we could have just turned it on by default and not even have this flag. If num worker is set, meaning the user wants a parallel training like job, then it is set on; if not, default will be off or the infra default - no need to expose more ports than needed.